src/main/java/io/github/moulberry/notenoughupdates/listener/ItemTooltipListener.java

```diff
@@ -690,7 +690,7 @@ public void onItemTooltipLow(ItemTooltipEvent event) {
 					if (i - 2 < 0) {
 						break;
 					}
-					newTooltip.addAll(i - 1, petToolTipXPExtend(event));
+					if (Utils.getOpenChestName().endsWith("Pets")) newTooltip.addAll(i - 1, petToolTipXPExtend(event));//also appears in storage
```

Comment:
I don't have any major changes to add rn, so if someone ever finds the comment, they can remove it (it was just to remember what I noticed during testing)